src/js/modules/infragistics.ui.editors.js

```diff
@@ -10166,11 +10203,18 @@
 		// igCheckboxEditor public methods
 		value: function (newValue) {
 			/* Gets/Sets Current checked state/Value of the igCheckboxEditor that will be submitted by the HTML form.
-				1. If the 'value' option IS NOT defined, then 'value' method will match the checked state of the editor.
+				1. If the [value](ui.igcheckboxeditor#options:value) option IS NOT defined, then 'value' method will match the checked state of the editor.
 				This is a good option when the checkbox is intended to operate as a Boolean editor. In that case the return type is bool.
-				2. If the 'value' option IS defined, then 'value' method will return the 'value' option,
+				2. If the [value](ui.igcheckboxeditor#options:value) option IS defined, then 'value' method will return the [value](ui.igcheckboxeditor#options:value) option,
```

Comment:
It won't return the value option, but it will submit the value when the editor is checked and the form is submitted.